tests/foreman/ui/test_repository.py

```diff
@@ -156,38 +153,37 @@ def test_positive_create_as_non_admin_user(module_org, test_name):
 @pytest.mark.tier2
 @pytest.mark.upgrade
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_create_puppet_repo_same_url_different_orgs(session, module_prod):
+def test_positive_create_yum_repo_same_url_different_orgs(session, module_prod):
```

Comment:
Is this a useful substitution? More specifically, is this a valid issue for yum repos?